### PR TITLE
Prep for bidirectional mesh support.

### PR DESCRIPTION
**Pull request by schinkowski**
Adding script to create symlinks in the test projects.
Preparing interface for unity UI methods.

### CODE REVIEW

**Review by seandillon92 — APPROVED** [review]

File: Editor/DefaultAppUtility.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
+﻿using System.IO;
+using Microsoft.Win32;
+
+public static class DefaultAppUtility
```
Comment (sindharta) [inline Editor/DefaultAppUtility.cs]: Please make this into an internal class, unless we want to open this API to users.

File: Runtime/Scripts/IDCCLauncher.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
+﻿namespace Unity.MeshSync
+{
+    public enum RunMode
```
Comment (sindharta) [inline Runtime/Scripts/IDCCLauncher.cs]: Please make this into an internal class, unless we want to open this API to users.

File: Runtime/Scripts/IDCCLauncher.cs [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    {
+        GUI,
+        Background,
+        Console
+    }
+
+    /// <summary>
+    /// Interface for classes that handle DCC tool control from inside Unity.
+    /// </summary>
+    public interface IDCCLauncher
```
Comment (sindharta) [inline Runtime/Scripts/IDCCLauncher.cs]: Please make this into an internal class, unless we want to open this API to users.

File: Editor/DefaultAppUtility.cs (hunk 1 above)
Comment (schinkowski) [reply Editor/DefaultAppUtility.cs]: Done

File: Runtime/Scripts/IDCCLauncher.cs (hunk 3 above)
Comment (schinkowski) [reply Runtime/Scripts/IDCCLauncher.cs]: Done

**Pull request merged**